### PR TITLE
Adsinteractive : adapter usersync endpoint modified

### PR DESCRIPTION
**Pull request by balintvargha**

### CODE REVIEW

**Comment by onkarvhanumante** [comment]
@balintvargha  could you resolve PR conflicts

**Comment by balintvargha** [comment]
@onkarvhanumante done

File: static/bidder-info/adsinteractive.yaml [hunk 1]
```diff
@@ -11,6 +11,6 @@ capabilities:
       - banner
 userSync:
   redirect:
-    url: "http://pb.adsinteractive.com/getuid?{{.RedirectURL}}"
+    url: "http://bid.adsinteractive.com/getuid?{{.RedirectURL}}"
     userMacro: "$AUID"
- 
\ No newline at end of file
+
```
Comment (onkarvhanumante) [inline static/bidder-info/adsinteractive.yaml]: verified. user sync is working

**Review by onkarvhanumante — APPROVED** [review]

File: static/bidder-info/adsinteractive.yaml (hunk 1 above)
Comment (gargcreation1992) [reply static/bidder-info/adsinteractive.yaml]: verified.

**Review by gargcreation1992 — APPROVED** [review]

**Pull request merged**